iroha2/sys/iroha_core/smartcontracts/isi/error/__init__.py

```diff
@@ -1,4 +1,2 @@
-from .....rust import Enum, Struct, Tuple, Dict
```

Comment:
I don't know if we need these packages as is. The Pythonic way is to have almost all of the useful types in the package root, and this form of re-export is more rust-y in nature.